shopfloor/tests/test_zone_picking_set_line_destination_pick_pack.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2021 Camptocamp SA (http://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo_test_helper import FakeModelLoader
+
+from .test_zone_picking_base import ZonePickingCommonCase
+
+
+class ZonePickingSetLineDestinationPickPackCase(ZonePickingCommonCase):
+    """Tests set_line_destination when `pick_pack_same_time` is one
```

Comment:
```suggestion
    """Tests set_line_destination when `pick_pack_same_time` is on
```

Reply:
BTW, as from now `pick_pack` seems enabled by default, no need to adapt the existing tests with this option turn off?

Reply:
it's not on by default. It's on only when enabled on the menu item.